Makefile

```diff
@@ -4,6 +4,8 @@ VERBOSE=-vvv
 
 install:
 	poetry install
+	mkdir -p ./src/matrix_validator/resources
+	curl -L https://raw.githubusercontent.com/biolink/biolink-model/refs/heads/master/biolink-model.yaml -o ./src/matrix_validator/resources/biolink-model.yaml
```

Comment:
https://w3id.org/biolink/biolink-model.yaml is the extra safe purl, the other nice way to get the model yaml is by importing biolink-model as a package and pulling from importlib. This is great for having control of the version, but slightly more awkward to access the yaml. I'm also not sure that my suggestion for how to get the full prefixmap would actually work through importlib, unfortunately right now, but that's probably something we could fix in the biolink repo.

Reply:
This is removed in favor of reading from the biolink-model dependency and importlib